handbook/engineering/deployments/postgresql.md

```diff
@@ -1,25 +1,72 @@
-# PostgreSQL database tips
+# PostgreSQL deployments
 
-### Shell commands
+For deployments other than Cloud and Sourcegraph.com please use the information [here](https://docs.sourcegraph.com/admin/faq#how-do-i-access-the-sourcegraph-database) to access the database.
 
-These commands assume you're on a local machine, and trying to access the live systems. Also refer to the [deployment page's Kubernetes section](./kubernetes.md) for kubectl tips.
+## Sourcegraph.com specific
 
-### Sourcegraph.com specific
+We currently run two separate databases. The `sg-cloud` database is the primary database, and the code-intel team uses the `sg-cloud-code-intel`.
 
-You can also directly view database in [GCP](https://console.cloud.google.com/sql/instances?project=sourcegraph-dev)
+You can also directly view the database in [GCP](https://console.cloud.google.com/sql/instances?project=sourcegraph-dev)
 
-We currently run two seperate databases. The `sg-cloud` database is the primary database and the `sg-cloud-code-intel` is used by the code-intel team.
+We utilize the GCP utility of the [Cloud SQL Proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) to connect to our production databases. By default, our Cloud SQL databases are not accessible without this proxy.
 
-Use the cloud sql proxy to connect to the database
+For read-only access, there is also an option of using [BigQuery](
+https://console.cloud.google.com/bigquery?sq=527047051561:67f2616f4acb4b7cb3639e4a97e2f4aa
+) and their `EXTERNAL_QUERY` syntax.
```

Comment:
I think using BigQuery might deserve a section of its own, with the full `EXTERNAL_QUERY` here (did not realize that it was just a stand-in for a table), e.g. seeing:

```sql
SELECT * FROM EXTERNAL_QUERY("...", "SELECT name::text,created_at::text FROM repo LIMIT 5;");
```

would be a lot more self-explanatory. Seems a lot less setup than the alternative, so for UI-and-mouse noobs like me I would probably go for this more often than not 😛